### PR TITLE
Vba

### PR DESCRIPTION
**Pull request by tik9**

### CODE REVIEW

File: c++/c++-quiz.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
+## C++ LinkedIn Assesment Questions Answers
+
+#### Q1. What is printed from this code?
+
+    vector<int> v(22);
+    bool b= (v[6]);
+    - []intf("%d",!b);
+
```
Comment (Ebazhanov) [inline c++/c++-quiz.md:8]:
```suggestion
```

File: AngularJS/angularjs-quiz.md [hunk 2]
```diff
@@ -1,147 +1,153 @@
...
-    b) It is used to change the value of a control to a new value. You would call that method and pass in the new value for the form field. It even supports passing in an array of values that can be set over time. It is used to change the value of a control to a new value. You would call that method and pass in the new value for the form field. It even supports passing in an array of values that can be set over time.
-    c) It returns a Boolean based on if the value of the control is different from the value with which it was initialized. It returns a Boolean based on if the value of the control is different from the value with which it was initialized.
-    d) It is an observable that emits every time the value of the control changes, so you can react to new values and make logic decisions at that time. It is an null that emits every time the value of the control changes, so you can react to new values and make logic decisions at that time.
+#### Q12. What directive is used to link an <a> tag to routing?
 
-Q12. What directive is used to link an <a> tag to routing?
+    - [] routeTo
+    - [] routerLink <---Correct
```
Comment (Ebazhanov) [inline AngularJS/angularjs-quiz.md:144]:
```suggestion
    - [x] routerLink
```

File: c++/c++-quiz.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+    - [] typedef struct {
+    		bit sunday;
+    		bit monday;
+    		// more days
+    		bit friday;
+    		bit satarday;
+    		} weekdays;
+
+#### Q12. What is an lvalue?
+- []re>a) It's a constant expression , meaning an expression composed of constants and operations.
```
Comment (Ebazhanov) [inline c++/c++-quiz.md:130]: could you please take a look maybe there is something wrong? and we should remove `re>a)`
```
- []re>a) It
```

File: django/django-quiz.md [hunk 4]
```diff
@@ -1,333 +1,333 @@
...
-    - group(); Model
-    - validate(); Form
-    👍 - clean(); Field
+    - [] validate(); Model
+    - [] group(); Model
+    - [] validate(); Form
+    - [x] - clean(); Field
 
-Q4. A client wants their site to be able to load "Rick & Morty" episodes by number or by title—e.g., shows/3/3 or shows/picklerick. Which URL pattern do you recommend?
+#### Q4. A client wants their site to be able to load "Rick & Morty" episodes by number or by titleÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â½e.g., shows/3/3 or shows/picklerick. Which URL pattern do you recommend?
```
Comment (Ebazhanov) [inline django/django-quiz.md:22]: original does not have these special characters, could you please double-check

`â€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã`

File: django/django-quiz.md [hunk 5]
```diff
@@ -1,333 +1,333 @@
...
-    - Implement the not_used method in your middleware class.
-    - List the middleware beneath an entry of django.middleware.IgnoredMiddleware.
-    - Write code to remove the middleware from the settings in [app]/__init__.py.
+    - [x] - Raise MiddlewareNotUsed in the __init__ function of your middleware.
+    - [] Implement the not_used method in your middleware class.
+    - [] List the middleware beneath an entry of django.middleware.IgnoredMiddleware.
+    - [] Write code to remove the middleware from the settings in [app]/__init__.py.
 
-Q6. How do you turn off Django’s automatic HTML escaping for part of a web page?
+#### Q6. How do you turn off DjangoÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â½s automatic HTML escaping for part of a web page?
```
Comment (Ebazhanov) [inline django/django-quiz.md:36]: ƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â

File: django/django-quiz.md [hunk 6]
```diff
@@ -1,333 +1,333 @@
...
 
-    - Place that section between paragraph tags containing the autoescape=off switch.
-    - Wrap that section between { percentage mark autoescape off percentage mark} and {percentage mark endautoescape percentage mark} tags.
-    - Wrap that section between {percentage mark autoescapeoff percentage mark} and {percentage mark endautoescapeoff percentage mark} tags.
-    👍 - You don't need to do anything—autoescaping is off by default.
+    - [] Place that section between paragraph tags containing the autoescape=off switch.
+    - [] Wrap that section between { percentage mark autoescape off percentage mark} and {percentage mark endautoescape percentage mark} tags.
+    - [] Wrap that section between {percentage mark autoescapeoff percentage mark} and {percentage mark endautoescapeoff percentage mark} tags.
+    - [x] - You don't need to do anythingÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â½autoescaping is off by default.
```
Comment (Ebazhanov) [inline django/django-quiz.md:41]: ƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â

File: vba/vba.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
+
```
Comment (Ebazhanov) [inline vba/vba.md:1]: I think you are adding a new quiz here, great but you also should update the main Readme by adding the reference link there in the table. Please remember that the order of the quizzes' names should be alphabetical (A-Z).

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@tik9 Thank you for formatting and adding new `Vba` quiz, 👍 -> left some comments! Please take a look when you have time.

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@tik9 there are some conflicts appears please take a look.

File: vba/vba.md (hunk 7 above)
Comment (tik9) [reply vba/vba.md:1]: ok, exactly Evgenii.

File: c++/c++-quiz.md (hunk 1 above)
Comment (tik9) [reply c++/c++-quiz.md:8]: Fixed, thanks for informing.

**Comment by tik9** [comment]
Should we close this PR?

**Comment by Ebazhanov** [comment]
as you wish, if you think it duplicates the existing one please close it FYI @tik9

**Pull request closed without merging**